systemtest/src/test/java/io/strimzi/systemtest/RollingUpdateST.java

```diff
@@ -342,14 +341,18 @@ void testKafkaWontRollUpBecauseTopic() {
     }
 
     @Test
-    @Disabled("Zookeeper scaleUp/scaleDown is currently covered by manual procedure for Kafka 2.4.x. " +
-            "This should be removed after implementation of Zookeeper Dynamic configuration")
     void testZookeeperScaleUpScaleDown() {
         int messageCount = 50;
         String topicName = "test-topic-" + new Random().nextInt(Integer.MAX_VALUE);
 
         timeMeasuringSystem.setOperationID(timeMeasuringSystem.startTimeMeasuring(Operation.CLUSTER_RECOVERY));
-        KafkaResource.kafkaPersistent(CLUSTER_NAME, 3).done();
+        KafkaResource.kafkaPersistent(CLUSTER_NAME, 3)
+                .editSpec()
+                    .editKafka()
+                        .withVersion("2.3.1")
```

Comment:
We change Kafka version by env variable `ST_KAFKA_VERSION`. If a specific version of kafka is needed, we should have a different test for it, because this one should be general for ZK scaleup/Scaledown

Reply:
That was actually commited by mistake.